app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,63 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+# before_filter :configure_sign_up_params, only: [:create]
+# before_filter :configure_account_update_params, only: [:update]
+
+  # GET /resource/sign_up
+  def new
+    @usertypes = Usertype.where("name != 'Administrator'")
+    super
+  end
+
+  # POST /resource
+  def create
+    configure_sign_up_params
+    super
+  end
+
+  # GET /resource/edit
+  def edit
+    @usertypes = Usertype.where("name != 'Administrator'")
+    super
+  end
+
+  # PUT /resource
+  def update
+    super
+  end
+
+  # DELETE /resource
+  def destroy
+    super
+  end
```

Comment:
then again, devise could be doing something else when running these methods. try removing them, but you might need them depending on how devise works